stable/traefik/templates/configmap.yaml

```diff
@@ -38,6 +38,7 @@ data:
         [entryPoints.http.auth.forward]
         address = {{ .Values.forwardAuth.address | quote }}
         trustForwardHeader = {{ .Values.forwardAuth.trustForwardHeader }}
+        authResponseHeaders = {{ .Values.forwardAuth.authResponseHeaders }}
```

Comment:
Have you tried testing the chart locally?

I templated the chart with the values...

```yaml
forwardAuth:
  entryPoints:
    - http
  
  authResponseHeaders:
    - "X-Forwarded-For"
```

and received the following output for `authResponseHeaders` in the configuration map.

```
...
authResponseHeaders = [X-Forwarded-For]
```

as per documentation https://docs.traefik.io/configuration/entrypoints/#forward-authentication, don't the `authResponseHeaders` values have to be quoted?

https://gyazo.com/d3beaf1a30874af9675c441a2615522f

Reply:
@KadenLNelson  it is an array so you should template it as `authResponseHeaders: ["X-Forwarded-For"]`

Reply:
@KadenLNelson I was missing some changes to handle the array in a better way. Give it a try again.